weblate/addons/models.py

```diff
@@ -208,34 +209,36 @@ def handle_addon_error(addon, component):
         addon.disable()
 
 
+def handle_addon_event(sender, component, translation, event_type, store):
+    """Executing add on event for particular event type."""
+    event_string = EVENT_STRING.get(event_type)
+    if event_string:
```

Comment:
I'd let the code fail instead of silently swallow errors here:

```suggestion
    event_string = EVENT_STRING[event_type]
```

Reply:
Done